docs/site/Validation-REST-layer.md

```diff
@@ -175,3 +175,84 @@ of 10 digits separated by `-` after the 3rd and 6th digits.
 ```
 
 {% include tip.html content="RegExp can be converted into a string with `.source` to avoid escaping backslashes" %}
+
+## Customize validation errors
+
+Since the error is being caught at the REST layer, the simplest way to customize
+the errors is to customize the
+[sequnce](https://loopback.io/doc/en/lb4/Sequence.html). It exists in all
+LoopBack applications scaffolded by using the `lb4` command and can be found in
+`src/sequence.ts`.
+
+Let's take a closer look at how to customize the error. A few things to note in
+the below code snippet:
+
+1. inject RestBindings.SequenceActions.LOG_ERROR for logging error and
+   RestBindings.ERROR_WRITER_OPTIONS for options
+2. customize error for particular endpoints
+3. create a new error with customized properties
+4. log the error using RestBindings.SequenceActions.LOG_ERROR
+
+{% include code-caption.html content="/src/sequence.ts" %}
+
+```ts
+export class MySequence implements SequenceHandler {
+  // 1. inject RestBindings.SequenceActions.LOG_ERROR for logging error
+  // and RestBindings.ERROR_WRITER_OPTIONS for options
+  constructor(
+    /*..*/
+    @inject(RestBindings.SequenceActions.LOG_ERROR)
+    protected logError: LogError,
+    @inject(RestBindings.ERROR_WRITER_OPTIONS, {optional: true})
+    protected errorWriterOptions?: ErrorWriterOptions,
+  ) {}
+
+  async handle(context: RequestContext) {
+    try {
+      // ...
+    } catch (err) {
+      this.handleError(context, <HttpErrors.HttpError>err);
```

Comment:
I think we prefer to use `as Type` instead of `<Type>` syntax for explicit type-casts.

```suggestion
      this.handleError(context, err as HttpErrors.HttpError);
```